planner/core/prepare_test.go

```diff
@@ -2758,6 +2775,11 @@ func TestPartitionWithVariedDataSources(t *testing.T) {
 			}
 		}
 	}
+	// Allow ~1% non-cached queries, due to background changes etc.
+	// (Actually just 1/3 %, since there are 3 tables * 4 queries per loop :)
+	if len(missedPlanCache) > (loops * 4 / 100) {
```

Comment:
`len(missedPlanCache)` looks like we only need a simple integer counter?

Reply:
Yes, but using the `[]string` array, we can also show all the different tables/plans it failed, so I think the combined use is ok.